<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="plan_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>711</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>일정 내용</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>카테고리</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>수행 여부</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>일정 내용</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="todo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>카테고리</string>
   </property>
  </widget>
  <widget class="QComboBox" name="plan_category">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>식단</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>운동</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>공부</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>오락</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>돈</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>청소</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>수면</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>외출</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>날짜</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>7</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>시간</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="end_time">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>추가하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>오늘의 일정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>업데이트하기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="count_lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
